addons/fluxcd/definitions/helm-release-def.yaml

```diff
@@ -57,6 +57,7 @@ spec:
         		name: context.name
         	}
         	spec: {
+        		timeout: parameter.installTimeout
```

Comment:
```suggestion
        		timeout: parameter.timeout
```

When we talk about `helm`, we mean installing most of the time. So `timeout` can clearly express the install context.

Reply:
Or it seems conflicts with the existing timeout.

Reply:
There's conflict yeah.